nix/sources.json

```diff
@@ -1,14 +1,14 @@
 {
     "nixpkgs": {
-        "branch": "hls-hlint-plugin-workaround",
+        "branch": "nixpkgs-unstable",
         "description": "Wire's fork of NixOS/nixpkgs. Use until HLS > 1.7.0.0 is available in NixOS/nixpkgs",
```

Comment:
This description sounds incorrect now, please fix.

Reply:
Done.